### ISSUE TITLE
model.py - class EfficientDet - self.conv5 question

### ISSUE THREAD

**Issue by RobertHong1992** [signatrix/efficientdet#51]
self.conv5 = nn.Conv2d(192, self.num_channels, kernel_size=1, stride=1, padding=0)
with paper , is  112?!
